contracts/Cross-chain/OmnichainGovernanceExecutor.sol

```diff
@@ -239,6 +239,23 @@ contract OmnichainGovernanceExecutor is ReentrancyGuard, BaseOmnichainController
         }
     }
 
+    /**
+     * @notice Resends a previously failed message.
+     * @param srcChainId_ Source chain Id.
+     * @param srcAddress_ Source address => local app address + remote app address.
+     * @param nonce_ Nonce to identify failed message
+     * @param payload_ The payload of the message to be retried.
+     * @custom:access Only owner.
+     */
+    function retryMessage(
+        uint16 srcChainId_,
+        bytes calldata srcAddress_,
+        uint64 nonce_,
+        bytes calldata payload_
+    ) public payable override onlyOwner nonReentrant {
```

Comment:
Is it not enough with `whenNotPaused` in `_nonblockingLzReceive`?

Reply:
The idea is to store the failed message in `OmnichainGovernanceExecutor` if the contract is paused. The implementation of `whenNotPaused` in `_nonblockingLzReceive` prevents us from retrying the unsuccessful message when the bridge is paused.

Reply:
I mean, do we need the `onlyOwner` modifier here?

Reply:
I see. It's related to https://github.com/VenusProtocol/governance-contracts/pull/64/files#r1560408174